### ISSUE TITLE
Generated webapp sass grunt task fails

### ISSUE THREAD

**Issue by phairoh** [yeoman/generator-webapp_DEPRECATED#568]
A [recent change](https://github.com/yeoman/generator-webapp/commit/1ed82fedbdbba664ebac2072caa75842a9e80b57#commitcomment-12018687) to the template Gruntfile.js seems to have broken the sass task of the generated web app.  The following error is generator whenever a task is run involving the sass task:

```
$ Warning: Running "sass:dist" (sass) task
$ file to import not found or unreadable: bootstrap-sass/assets/stylesheets/_bootstrap.scss
$ Current dir: <path to webapp>/app/styles/
$ Line 4  Column 9  app\styles\main.scss
$ Warning:  Use --force to continue.
```

This error has been seen both on Windows 7 and OS-X 10.10.4.

As mentioned in the comment, reverting the change to the template Gruntfile.js seems to fix the issue.

**Comment by silvenon** [comment]
Thanks for reporting and providing a fix. I forgot to reflect the wiredep changes in `Gruntfile.js`, like I did in `index.js`, so I decided to go that way instead of reverting.

**Comment by silvenon** [comment]
Released in [v1.0.1](https://github.com/yeoman/generator-webapp/releases/tag/v1.0.1).
